.github/workflows/zips.yml

```diff
@@ -31,9 +31,7 @@ jobs:
           restore-keys: |
             ${{ runner.os }}-composer-
       - name: Composer Install
-        uses: php-actions/composer@v1
-        with:
-          args: install
+        run: composer install --no-interaction --no-suggest --no-progress
```

Comment:
Why did this need to be updated?

Reply:
@felixarntz because `php-actions/composer` generates files owned by `root` user. It causes permission denied errors when we try to re-generate autoload map.

Reply:
@felixarntz I believe there was a permissions issue due to ownership differences between composer which runs in the action's container and the "local" composer since we now run composer during the zip command which uses the "local" composer binary and user permissions.